### PR TITLE
Add 'primitive', 'savage', 'sophisticated', 'tribe', 'tribal', 'stone…

### PR DESCRIPTION
**Pull request by benmarwick**
… age'

Following up on the discussion at https://github.com/retextjs/retext-equality/issues/70

### CODE REVIEW

File: script/race.yml [hunk 1]
```diff
@@ -141,3 +141,31 @@
     - allow list
   inconsiderate:
     - whitelist
+- type: simple
+  note: Avoid using terms that imply a group has not changed over time and that they are inferior
+    - simple
```
Comment (wooorm) [inline script/race.yml]: You’re missing `considerate` here:

```suggestion
  considerate:
    - simple
```

Comment (benmarwick) [reply script/race.yml]: Oops, my mistake! Thanks for spotting that

**Comment by wooorm** [comment]
Oh, I forgot, could you also run [`npm test`](https://github.com/retextjs/retext-equality/blob/7cb2d87897305e477232595f0fb70ee18e9e0b09/package.json#L84)? It updates the list of rules with these new patterns too

**Comment by benmarwick** [comment]
Yes, no worries. All tests passed on OSX:

```
1..54
# tests 54
# pass  54
```

**Comment by wooorm** [comment]
Ah, sorry, I meant that running `npm test` also updates files to reflect your changes. Sorry, should’ve clarified that.

Anyway, landed, thank you! 👍

**Comment by benmarwick** [comment]
Got it, thanks! Looking forward to using this with my undergrad students to help then avoid writing things that they usually lose points for.

**Pull request closed without merging**